docker/Readme.md

```diff
@@ -0,0 +1,86 @@
+# Docker for server
+
+## build
+The docker is easyly build using the following command - By default it will build the docker for the **develop** branch:
```

Comment:
The docker is easily built using the following command - By default, it builds the docker for the **develop** branch:

Reply:
Ok

Reply:
please check it again...

Reply:
Copy-pasted your "text"